examples/forward/BUILD

```diff
@@ -0,0 +1,100 @@
+cc_test(
```

Comment:
Is there an automatic approach that will work any time a new example file is added? This file may get forgotten when the list of examples gets expanded over time.

Reply:
I am not aware of such a functionality. Will have a look if it is possible to generate/group these automatically.